README.md

```diff
@@ -192,6 +192,8 @@ To create an image based on Ubuntu Xenial for Raspberry Pi 2 with [htop](https:/
 $ sudo env DEVICE=rpi-2-b OS=ubuntu-xenial-armhf INCLUDES=htop,mc ./pieman.sh
 ```
 
+Notice, that you can also specify the parameters in `.env` file. In this case they will have the higher priority then arguments specified as command line arguments.
```

Comment:
Let us be consistent -- there are a lot of parts with `Note that...` in this README.

Reply:
`than` instead of `then`.